.gitignore

```diff
@@ -24,3 +24,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```

Comment:
Please use yarn instead of npm, or decide at project level what you guys want to use. If is npm please remove packge-lock.json from commited files as well

Reply:
We're using yarn, so i deleted "npm-debug.log*"